performance-tests/commoncrawl-project/steps/mapping/MapWat.step.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name" : "MapWat",
+  "stepDefinitionName" : "entity-services-mapping",
+  "stepDefinitionType" : "MAPPING",
+  "sourceQuery" : "cts.andQuery([cts.collectionQuery([\"WebPage\"]), cts.jsonPropertyValueQuery(\"WARC-Type\", \"response\")])",
+  "sourceDatabase" : "data-hub-STAGING",
+  "collections" : [ "MapWat", "WebSite" ],
+  "permissions" : "data-hub-operator,read,data-hub-operator,update",
+  "targetDatabase" : "data-hub-FINAL",
+  "targetFormat" : "json",
+  "targetEntityType" : "http://marklogic.com/cc/WebSite-0.0.1/WebSite",
+  "properties" : {
+    "opengraph_id" : {
+      "sourcedFrom" : "//HTML-Metadata/twitter_site"
+    },
+    "server" : {
+      "sourcedFrom" : "normalizeServer(//Headers/Server)"
```

Comment:
This is a pretty complex mapping step with all these custom functions - we'll likely slim this down to something with no mapping functions in the future, but it's fine for now. We'll definitely want a complex one like this too.